### PR TITLE
[RTM] update docs on patching containers

### PR DESCRIPTION
**Pull request by chrisfilo**
- Fixed paths
- Removed egg creation part (not necessary anymore)
- added paragraph on Singularity

### CODE REVIEW

File: docs/contributors.rst [hunk 1]
```diff
@@ -24,54 +24,34 @@ In order to test new code without rebuilding the Docker image, it is
 possible to mount working repositories as source directories within the
 container.
 In the docker container, the following Python sources are kept in
```
Comment (effigies) [inline docs/contributors.rst]: `s/the following Python sources are kept/all Python packages are installed/`

File: docs/contributors.rst [hunk 2]
```diff
@@ -24,54 +24,34 @@ In order to test new code without rebuilding the Docker image, it is
...
-``/root/src``: ::
-
-    /root/src
-    ├── fmriprep/
-    ├── nipype/
-    └── niworkflows/
+``/usr/local/miniconda/lib/python3.6/site-packages``.
 
 To patch in working repositories, for instance contained in
```
Comment (effigies) [inline docs/contributors.rst]: working repositories *of FMRIPREP or its dependencies*

File: docs/contributors.rst [hunk 3]
```diff
@@ -24,54 +24,34 @@ In order to test new code without rebuilding the Docker image, it is
...
-        poldracklab/fmriprep:latest
-    root@03e5df018c5e:~# cd ~/src/fmriprep/
-    root@03e5df018c5e:~/src/fmriprep# pip install -e .
-    root@03e5df018c5e:~# cd ~/src/niworkflows/
-    root@03e5df018c5e:~/src/niworkflows# pip install -e .
-    root@03e5df018c5e:~# cd ~/src/nipype/
-    root@03e5df018c5e:~/src/nipype# pip install -e .
+Patching containers can be achieved in Singularity by using the PYTHONPATH variable: ::
+
+   $ PYTHONPATH="$HOME/projects/fmriprep/fmriprep" singularity run fmriprep.img \
```
Comment (effigies) [inline docs/contributors.rst]: Safest to suggest:

`PYTHONPATH="$HOME/projects/fmriprep:$PYTHONPATH"`

(Removed one of the `fmriprep`s in that path... I think that's probably what you meant.)

Comment (chrisfilo) [inline docs/contributors.rst]: You probably don't want other packages in your pythonpath to interfere...

Comment (effigies) [inline docs/contributors.rst]: That's fair. And if somebody knows they want it, they know enough to update the env var.

**Pull request merged**